src/mvGlobalHandlers.cpp

```diff
@@ -273,7 +273,7 @@ void mvMouseDoubleClickHandler::draw(ImDrawList* drawlist, float x, float y)
 	{
 		for (int i = 0; i < IM_ARRAYSIZE(ImGui::GetIO().MouseDown); i++)
 		{
-			if (ImGui::IsMouseDoubleClicked(i))
+			if (ImGui::GetMouseClickedCount(i) == 2)
```

Comment:
Is there any reason to use `GetMouseClickedCount()` here rather than `IsMouseDoubleClicked()`? They are almost identical, except for cases where `IsMouseDoubleClicked()` can be affected by key locking, which is not used in DPG and ImGui anyway. `IsMouseDoubleClicked()` reads better, so why change it?

Reply:
The `GetMouseClickedCount()` is a new function implemented in ImGui and I thought that `IsMouseDoublClicked()` would have became soon deprecated so, in order to prevent future changes, I changed it.
No real reason other than this one, so you tell me if revert it or keep it like it is now.

Reply:
I don't think there's a reason for `IsMouseDoubleClicked()` to get deprecated. People seem to be using it a lot (judging by comments in ImGui tickets), it's been a part of the public API since before the dinosaurs, and it reads much better than `GetMouseClickedCount() == 2`. So my vote goes for leaving `IsMouseDoubleClicked()` in place.